### ISSUE TITLE
advice on optimal use of slidy for presentations

### ISSUE THREAD

**Issue by OskaratOmni** [rfortherestofus/omni#55]
Hey @dgkeyes  and/or @tvroylandt 
I am working on a presentation with the slidy template. It has some nice features, but sometimes I struggle to implement basic formatting features. 
For instance, I have a short outline for a talk and it would be great to display this over two columns. I used 
<div class="columns-2">
- Topic 1
  - Bullet 1
  - Bullet 2
  - Bullet 3

- Topic 2
  - Bullet 1
  - Bullet 2
  - Bullet 3
</div>

from the rmarkdown book hoping it would work (it is perhaps for ioslides only ), but it did not. 

And things like having the bullets iteratively appear works okay but only if I don't use the template. As far as I can tell using the template prevents changing options in the yaml. Is that right? 

I don't think we should spend much time making an amazing and feature-rich template. It isn't that important. I am looking for a quick and easy way to add a few things. 

Do you have some thoughts on easy way to use slidy? 
I know that quarto has a presentation format, and there are others in markdown too. 

Or should I embrace the lack of features in slidy as a 'feature' rather than a bug bc it forces some simplicity on the output, which can sometimes be a good thing. :-) 
(if I can't add columns or other features it is really not a big problem)

**Comment by tvroylandt** [comment]
I'm not really a Slidy expert but the template in the `omni` package shouldn't prevent to use any option in the YAML.

The only thing is that you don't have a lot of features. You can use HTML one (see https://stackoverflow.com/questions/58243552/two-column-slide-in-r-slidy-presentation for columns as an example).

**Comment by OskaratOmni** [comment]
Oh yes of course, thanks. I really should have thought of that as I have used it for bookdown and markdown outputs before. 
I'll go with this for now but maybe will return to the yaml question later on so please keep open for a little longer

**Comment by dgkeyes** [comment]
I'm afraid I don't know anything about slidy either. I've used `xaringan` extensively and it allows for both columns and incremental reveal ([see Chapter 8 of my book](https://book.rwithoutstatistics.com/presentations-chapter)).

**Comment by OskaratOmni** [comment]
Yeah and it may be that xaringan is a better choice. It seems to have more of a user base and is trying to have some of the elements that were common beamer presentations. 

So when I say that the yaml settings work differently when I use the template, this is what I mean...
In using a slidy directly from Rmarkdown (no template) I can set the yaml like this and get incremental reveal: 

---
title: "Elevate Boulder: Cash Assistance for Financial Stability"
subtitle: "Procedure for sample weighting and selection"
author: "OMNI Institute"
date: "`r Sys.Date()`"
output: 
  slidy_presentation:
    incremental: true
    font_adjustment: +1
---

but doing something like this does not

---
title: "Elevate Boulder: Cash Assistance for Financial Stability"
subtitle: "Procedure for sample weighting and selection"
author: "OMNI Institute"
date: "`r Sys.Date()`"
output: 
  omni::omni_slidy
    incremental: true

I have tried putting the incremental: true in different places and it always returns and error for me. 

Again, not worth it to spend much time on this but if you know a simple fix please let me know. 
Maybe next year after these more pressing resources are developed we can consider making an xaringan template.

**Comment by dgkeyes** [comment]
I don't know of a quick fix here. @tvroylandt any ideas?

**Comment by tvroylandt** [comment]
On my side, this works and the incremental option also works

```
title: "Elevate Boulder: Cash Assistance for Financial Stability"
subtitle: "Procedure for sample weighting and selection"
author: "OMNI Institute"
date: "r Sys.Date()"
output:
  omni::omni_html_slidy:
    incremental: true
```

Maybe it's a matter of having `:` after the format ?

**Comment by tvroylandt** [comment]
Good. May I close the issue ?

**Comment by OskaratOmni** [comment]
One more question @tvroylandt  - do you know how to have a section break slide in Slidy? 

Imagine I have three major topics in my talk, and when I move from one topic to the other, I would like a slide that is similar to the title slide and just has text in large format centered that says the name of the next section. - something like that. 

I can make my own work around here, but if you have an idea for how to do this easily please let me know. 
No problem if nothing comes to mind. Otherwise we can close the issue. thanks!

**Comment by tvroylandt** [comment]
@OskaratOmni As in other HTML format, you can setup a class for the slide like that 
```
## Section slide {.highlight}
```

Then you have to provide a custom CSS (or inside the document with style) . Please note that you should use `!important` to have the style apply on top of the default one.

```
<style>
.highlight{
  background: red !important;
}
</style>
```

**Comment by OskaratOmni** [comment]
Thanks @tvroylandt  this is working well. 
I know very little html (or css) but this was easy to modify. 
The way I have it now is working. 
This is how I have it: 

```
## {.highlight}

<span align="center"> <h1> New Section! </h1> </span>
```

I think you can see how this approach could work, but it is only the first thing I thought of. 

Two additional features that would be nice, but not critical, to add would be: 

1 - have the text centered both horizontally and vertically. The above code centers only horizontally. Is that easy to do? If not, keeping it placed as an h1 header against a shaded background definitely works. 

2 - rather than using <h1> is there an easy way to modify the above code such that the font is larger than the document h1? 

If those are difficult or annoying to add just let me know. But I suspect it could be quite easy for you to adjust the above to make it a little better. 

Thanks again!

**Comment by tvroylandt** [comment]
@OskaratOmni just to tell you I don't forgot this issue (thanks to David). I'm off a few days and will add that after

**Comment by tvroylandt** [comment]
You can style directly the element you want with :

```

<style>
.highlight{
  background: red !important;
}

.highlight > h1 {
  color: white ;
  font-size: 36pt!important;
  text-align: center!important;
  padding-top: 30%!important;
}
</style>
```

In fact, if you know what you want in terms of style, we can put that in a CSS file so that it would be transparent.
Here it's mandatory to force the property with `!important`, which isn't always the case
